tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -3245,8 +3245,11 @@ node_modules/chrome-devtools-frontend/front_end/bindings/ResourceScriptMapping.j
 node_modules/chrome-devtools-frontend/front_end/bindings/ResourceScriptMapping.js(298,22): error TS2694: Namespace 'Common' has no exported member 'Event'.
 node_modules/chrome-devtools-frontend/front_end/bindings/ResourceScriptMapping.js(305,22): error TS2694: Namespace 'Common' has no exported member 'Event'.
 node_modules/chrome-devtools-frontend/front_end/bindings/ResourceScriptMapping.js(318,26): error TS2694: Namespace 'Protocol' has no exported member 'Runtime'.
+node_modules/chrome-devtools-frontend/front_end/bindings/ResourceScriptMapping.js(323,14): error TS2339: Property '_scriptSource' does not exist on type 'ResourceScriptFile'.
```

Comment:
`this` is now typed, but we can't resolve the properties on the type (it may be incorrect).